Sources/Build/Command.swift

```diff
@@ -29,12 +29,16 @@ struct Target {
     /// to a client wanting to control the build.
     let name: String
 
-    /// A list of commands to run when building the target.  A command may be
+    /// A list of outputs that represent the target.
+    var outputs: [String]
```

Comment:
Isn't this also redundant? Looks like it is always the same as `cmds.flatMap{ $0.tool.outputs }`?